### ISSUE TITLE
'Observable<{}>' is not assignable to type 'Observable<ToastData> under beta 13 and zone 0.6.8

### ISSUE THREAD

**Issue by mburger81** [akserg/ng2-toasty#6]
Hey, using ng2-toasty 1.3.0 with angular2 beta.13 and zone 0.6.8 I get this error

```
node_modules/ng2-toasty/src/toasty.service.ts:59:8 
Type 'Observable<{}>' is not assignable to type 'Observable<ToastData>'.
  Type '{}' is not assignable to type 'ToastData'.
    Property 'id' is missing in type '{}'.
```

**Comment by akserg** [comment]
I will update it only today. Let you know.

**Comment by taohd** [comment]
any updates on this? possible issue with rxjs@5.0.0-beta.2 which angular2 beta.13 still has a dependency on. notice your project was using rxjs@5.0.0-beta.4

**Comment by taohd** [comment]
just needed to type the Observable in toasty.service.ts
`this.toastsObservable = new Observable<ToastData>((subscriber:Subscriber<ToastData>) => {
            this.toastsSubscriber = subscriber;`

**Comment by akserg** [comment]
Sorry, I tested the component with following combinations and didn't find any issues:
- Angular 2 beta 13, 
- RxJS beta 2 and 4
- Zone 0.6.6 and 0.6.8

Here is the Plunker's link you can use for tests: http://embed.plnkr.co/5aQl8L.

I'm ready to help you. Is that possible to some how share the project, source code, package.json and etc.

**Comment by mburger81** [comment]
@akserg I think this is not a issue but iats a TSlint limitation? Do you use latest angular2-webpack-starter there you can see the "issue".

On change code to
`this.toastsObservable = new Observable<ToastData>((subscriber:Subscriber<ToastData>) => {`
in line 59 this resloves the tslint error

thx
  Michael

**Comment by akserg** [comment]
Fixed in release 1.4.1 available on https://www.npmjs.com/package/ng2-toasty
